bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -844,36 +844,46 @@ protected void runRule(RuntimeRule rule, RuleEngineCallbackImpl.TriggerData td)
         }
     }
 
+    protected void runNow(String ruleUID, boolean considerConditions){
+    	 RuntimeRule rule = getRuntimeRule(ruleUID);
+         if (rule == null) {
+             logger.warn("Fail to execute rule '{}': {}", ruleUID, "Invalid Rule UID.");
```

Comment:
I see that it is not from your PR, but could you please fix these?

typo: Failed

Don't make "Invalid Rule UID." a parameter, but add it to the logging string itself.